### DISCUSSION TITLE
Tem algum repositorio com grade de estudo para programação

### DISCUSSION THREAD

**Discussion by AtaideDev** [AtaideDev/test#1] [Q&A] [answered]
Eu vi que tem muitas tecnologias na área de desenvolvimento e gostaria de saber se tem alguma api ou alguem tem uma grade que eu possa estudar

**Comment by ResolverProblems** [comment, marked as answer]
Existe, nesse repositório <https://github.com/kamranahmedse/developer-roadmap>
